pkgs/top-level/unix-tools.nix

```diff
@@ -0,0 +1,131 @@
+{ pkgs, buildEnv, runCommand, hostPlatform }:
+
+# These are some unix tools that are commonly included in the /usr/bin
+# and /usr/sbin directory under more normal distributions. Along with
+# coreutils, these are commonly assumed to be available by build
+# systems, but we can't assume they are available. In Nix, we list
+# each program by name directly through this unixtools attribute.
+
+# You should always try to use single binaries when available. For
+# instance, if your program needs to use "ps", just list it as a build
+# input, not "procps" which requires Linux.
+
+let
+
+  singleBinary = cmd: providers:
+    if builtins.hasAttr hostPlatform.parsed.kernel.name providers then
+      runCommand cmd {} ''
```

Comment:
I'd like to do a passthru var here that points to "$out/bin/${cmd}". Does anyone know if this is possible (without placeholder).